loadtest/user/userentity/websocket.go

```diff
@@ -41,6 +57,12 @@ func (ue *UserEntity) wsEventHandler(ev *model.WebSocketEvent) error {
 		if _, err := ue.store.DeleteReaction(reaction); err != nil {
 			return err
 		}
+	case model.WEBSOCKET_EVENT_POSTED:
```

Comment:
You can just write them as comma separated:

````
case <a>, <b>, <c>: 
// handle it
```

Reply:
That's great :+1: